src/App.js

```diff
@@ -84,9 +84,10 @@ function App() {
           </Switch>
           <nav
             style={{
-              position: 'absolute',
+              position: 'sticky',
```

Comment:
Is this having the effect you'd like? With `position: absolute`, it will for sure appear at the bottom of the page. Just double checking.

Reply:
@dariatsvetkova Since we set the background color here to be white, should we switch this back to absolute positioning? That way the router links are always at the bottom of the page, not just when the shopping list is long enough to push it down